### PR TITLE
Perftest partial run

### PR DESCRIPTION
**Pull request by akhdrv**
В этом пр апается версия перфтула, активируется фича перфтула по частичному прогону и фиксится баг с установкой зависимостей при прогоне в базовой ветке (сейчас ни актуальный кэш не достается, ни зависимости заново не устанавливаются).

Протестировано в https://github.com/salute-developers/plasma/pull/647. Сейчас проверка падает, потому что в базовой ветке старая версия

Job - https://github.com/salute-developers/plasma/actions/runs/5814842082/job/15765323689

Screenshot 2023-08-10 at 16 52 56

Screenshot 2023-08-10 at 16 52 47

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.644.5811911919.swift'>color_default_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.644.5811911919.kt'>color_default_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.644.5811911919.ts'>color_default_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.644.5811911919.xml'>color_default_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.644.5811911919.swift'>color_plasma_b2c_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.644.5811911919.kt'>color_plasma_b2c_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.644.5811911919.ts'>color_plasma_b2c_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.644.5811911919.xml'>color_plasma_b2c_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.644.5811911919.swift'>color_plasma_web_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.644.5811911919.kt'>color_plasma_web_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.644.5811911919.ts'>color_plasma_web_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.644.5811911919.xml'>color_plasma_web_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.644.5811911919.swift'>color_sberHealth_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.644.5811911919.kt'>color_sberHealth_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.644.5811911919.ts'>color_sberHealth_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.644.5811911919.xml'>color_sberHealth_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.644.5811911919.swift'>color_sbermarket_business_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.644.5811911919.kt'>color_sbermarket_business_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.644.5811911919.ts'>color_sbermarket_business_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.644.5811911919.xml'>color_sbermarket_business_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.644.5811911919.swift'>color_sbermarket_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.644.5811911919.kt'>color_sbermarket_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.644.5811911919.swift'>color_sbermarket_metro_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.644.5811911919.kt'>color_sbermarket_metro_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.644.5811911919.ts'>color_sbermarket_metro_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.644.5811911919.xml'>color_sbermarket_metro_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.644.5811911919.ts'>color_sbermarket_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.644.5811911919.swift'>color_sbermarket_selgros_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.644.5811911919.kt'>color_sbermarket_selgros_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.644.5811911919.ts'>color_sbermarket_selgros_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.644.5811911919.xml'>color_sbermarket_selgros_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.644.5811911919.swift'>color_sbermarket_wlbusiness_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.644.5811911919.kt'>color_sbermarket_wlbusiness_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.644.5811911919.ts'>color_sbermarket_wlbusiness_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.644.5811911919.xml'>color_sbermarket_wlbusiness_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.644.5811911919.xml'>color_sbermarket_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.644.5811911919.swift'>color_sberonline_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.644.5811911919.kt'>color_sberonline_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.644.5811911919.ts'>color_sberonline_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.644.5811911919.xml'>color_sberonline_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.644.5811911919.swift'>color_sberprime_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.644.5811911919.kt'>color_sberprime_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.644.5811911919.ts'>color_sberprime_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.644.5811911919.xml'>color_sberprime_xml--canary.644.5811911919.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.644.5811911919.swift'>color_stylesSalute_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.644.5811911919.kt'>color_stylesSalute_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.644.5811911919.ts'>color_stylesSalute_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.644.5811911919.xml'>color_stylesSalute_xml--canary.644.5811911919.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.644.5811911919.ts'>shadow_default_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.644.5811911919.ts'>shadow_plasma_b2c_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.644.5811911919.ts'>shadow_plasma_web_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.644.5811911919.ts'>shadow_sberHealth_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.644.5811911919.ts'>shadow_sbermarket_business_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.644.5811911919.ts'>shadow_sbermarket_metro_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.644.5811911919.ts'>shadow_sbermarket_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.644.5811911919.ts'>shadow_sbermarket_selgros_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.644.5811911919.ts'>shadow_sbermarket_wlbusiness_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.644.5811911919.ts'>shadow_sberonline_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.644.5811911919.ts'>shadow_sberprime_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.644.5811911919.ts'>shadow_stylesSalute_react-native--canary.644.5811911919.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.644.5811911919.swift'>typo_mage_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.644.5811911919.kt'>typo_mage_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.644.5811911919.ts'>typo_mage_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.644.5811911919.swift'>typo_plasma_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.644.5811911919.kt'>typo_plasma_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.644.5811911919.ts'>typo_plasma_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.644.5811911919.swift'>typo_ruler_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.644.5811911919.kt'>typo_ruler_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.644.5811911919.ts'>typo_ruler_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.644.5811911919.swift'>typo_sage_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.644.5811911919.kt'>typo_sage_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.644.5811911919.ts'>typo_sage_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.644.5811911919.swift'>typo_sbermarket_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.644.5811911919.kt'>typo_sbermarket_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.644.5811911919.ts'>typo_sbermarket_react-native--canary.644.5811911919.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.644.5811911919.swift'>typo_soulmate_ios-swift--canary.644.5811911919.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.644.5811911919.kt'>typo_soulmate_kotlin--canary.644.5811911919.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.644.5811911919.ts'>typo_soulmate_react-native--canary.644.5811911919.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.169.1-canary.644.5811911919.0
  npm install @salutejs/plasma-ui@1.201.1-canary.644.5811911919.0
  # or 
  yarn add @salutejs/plasma-temple@1.169.1-canary.644.5811911919.0
  yarn add @salutejs/plasma-ui@1.201.1-canary.644.5811911919.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-644/
ui storybook: http://plasma.sberdevices.ru/pr/pr-644/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-644/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-644/
ui storybook: http://plasma.sberdevices.ru/pr/pr-644/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-644/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

Top 5 significant rerender changes:
packages/plasma-ui/src/components/Pickers/Pickers.perftest.tsx#DatePicker: -4113.36%
packages/plasma-ui/src/components/Pickers/Pickers.perftest.tsx#TimePicker: -2761.77%
packages/plasma-b2c/src/components/PreviewGallery/PreviewGallery.perftest.tsx#AddButton: -1254.10%
packages/plasma-b2c/src/components/PreviewGallery/PreviewGallery.perftest.tsx#Draggable: -1121.10%
packages/plasma-web/src/components/TextField/TextField.perftest.tsx#Default: -1044.79%
Top 5 significant render changes:
packages/plasma-b2c/src/components/Calendar/Calendar.perftest.tsx#DoubleRange: -36.70%
packages/plasma-web/src/components/Calendar/Calendar.perftest.tsx#DoubleRange: -33.86%
packages/plasma-b2c/src/components/Calendar/Calendar.perftest.tsx#Double: -26.58%
packages/plasma-web/src/components/Calendar/Calendar.perftest.tsx#Double: -25.45%
packages/plasma-b2c/src/components/Calendar/Calendar.perftest.tsx#Default: -18.30%

**Comment by Yeti-or** [comment]
Может это еще `Draft` ?

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-644/
ui storybook: http://plasma.sberdevices.ru/pr/pr-644/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-644/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-644/

**Review by Yakutoc — APPROVED** [review]

File: .github/workflows/performance-test-pr.yml [hunk 1]
```diff
@@ -87,13 +87,47 @@ jobs:
...
+          s3://${{ secrets.AWS_S3_BUCKET_2 }}/perftool-cache/
+
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.base.ref }}
           clean: false
 
+      - name: Cache node modules
+        id: node_modules_base
+        uses: actions/cache/restore@v3
```
Comment (Yakutoc) [inline .github/workflows/performance-test-pr.yml:111]: [Уточнение]

@akhdrv Привет 

Здесь ты только достаешь `cache`, а если он не найден? 

Где логика на его создания?

Comment (akhdrv) [reply .github/workflows/performance-test-pr.yml:111]: из базовой ветки должен доставаться. а вообще, от предыдущего шага кэша в конце воркфлоу как раз этот кэш должен сохраниться

Comment (Yakutoc) [reply .github/workflows/performance-test-pr.yml:111]: Я так понял когда мы используем `actions/cache@v3` там механизм взять/сохранить есть. 

И ты правильно заметил что в конце workflow, например этот блок  

```
      - name: Cache node modules
        id: node_modules
        uses: actions/cache@v3
        with:
          path: node_modules
          key: node-modules-${{ hashFiles('package-lock.json') }}
          restore-keys: |
            node-modules-${{ hashFiles('package-lock.json') }}
            node-modules-
```
сохранит cache если он не будет найден. 

-----

actions/cache/restore@v3 - тут другая история   

> Once the cache is restored, unlike actions/cache, this action won't run a post step to do post-processing, and the rest of the workflow will run as usual.

Вот поэтому и спросил у тебя )

Comment (akhdrv) [reply .github/workflows/performance-test-pr.yml:111]: Я посчитал, что сохранять кэш зависимостей базовой ветки в ПРном воркфлоу не очень корректно, хотя вроде противоречий нет)

> The cache is scoped to the key, version, and branch. The default branch cache is available to other branches.

Получается этот кэш если создастся, полезен он будет только для текущей ветки, для последующих прогонов только этой джобы. Давай добавим, если не помешает

**Pull request merged**